OTVision/detect/detect.py

```diff
@@ -73,6 +74,7 @@ def main(
     chunksize: int = CONFIG[DETECT][YOLO][CHUNK_SIZE],
     half_precision: bool = CONFIG[DETECT][HALF_PRECISION],
     force_reload_torch_hub_cache: bool = CONFIG[DETECT][FORCE_RELOAD_TORCH_HUB_CACHE],
+    skip_validation: bool = CONFIG[DETECT][SKIP_VALIDATION],
```

Comment:
Added this as a config option. It might be necessary outside of tests, when we run a lot of jobs on a cluster which has only one public ip adress visible to github.